### PR TITLE
sse3

### PR DESCRIPTION
**Pull request by gwenn**
Complete.

### CODE REVIEW

File: src/x86/sse2.rs [hunk 1]
```diff
@@ -1762,6 +1762,16 @@ pub unsafe fn _mm_store_pd(mem_addr: *mut f64, a: f64x2) {
     *(mem_addr as *mut f64x2) = a;
 }
 
+/// Load a double-precision (64-bit) floating-point element from memory
+/// into both elements of returned vector.
+#[inline(always)]
+#[target_feature = "+sse2"]
+#[cfg_attr(test, assert_instr(movddup))]
```
Comment (BurntSushi) [inline src/x86/sse2.rs]: I've found that testing specific instructions for intrinsics that are basically just moves isn't too reliable. The specific mov variants seems to differ across targets, so I've just been leaving off tests for those.

Comment (BurntSushi) [reply src/x86/sse2.rs]: (And yeah, it looks like this is causing CI to fail on Linux.)

Comment (gwenn) [reply src/x86/sse2.rs]: Fixed.

**Comment by BurntSushi** [comment]
Sweet!

**Pull request merged**